code/game/machinery/cryopod.dm

```diff
@@ -278,6 +278,8 @@ GLOBAL_VAR_INIT(cryopods_enabled, FALSE)
 				to_chat(M.current, "<BR>[span_userdanger("Your target is no longer within reach. Objective removed!")]")
 				M.announce_objectives()
 		else if(O.target == mob_occupant.mind)
+			if((O.type in typesof(/datum/objective/assassinate)) && O.check_completion()) //kill once/kill+clone objective that's already been completed, don't give a new objective
```

Comment:
If you want it to apply to all assasinate and subtypes. (Like you already were)
```suggestion
			if((istype(O, /datum/objective/assassinate)) && O.check_completion()) //kill once/kill+clone objective that's already been completed, don't give a new objective
```
OR, if you only want subtypes
```suggestion
			if((O.type in subtypesof(/datum/objective/assassinate)) && O.check_completion()) //kill once/kill+clone objective that's already been completed, don't give a new objective
```

Reply:
oh wow, just checking subtypes, this goes in my folder of useful things to know